salt/runners/venafiapi.py

```diff
@@ -0,0 +1,515 @@
+        )
+    return data.get('dict', {})
+
+
+def show_csrs():
+    '''
+    Show certificate requests for this API key
+    '''
+    data = salt.utils.http.query(
+        '{0}/certificaterequests'.format(base_url),
+        status=True,
+        decode=True,
+        decode_type='json',
+        header_dict={
+            'tppl-api-key': _api_key(),
+        },
+    )
+    status = data['status']
+    if str(status).startswith('4') or str(status).startswith('5'):
+        raise CommandExecutionError(
+            'There was an API error: {0}'.format(data['error'])
+        )
+    return data.get('dict', {})
+
+
+def show_zones(company_id):
+    '''
+    Show certificate requests for this API key
+    '''
+    data = salt.utils.http.query(
+        '{0}/companies/{1}/zones'.format(base_url, company_id),
+        status=True,
+        decode=True,
+        decode_type='json',
+        header_dict={
+            'tppl-api-key': _api_key(),
+        },
+    )
+    status = data['status']
+    if str(status).startswith('4') or str(status).startswith('5'):
+        raise CommandExecutionError(
+            'There was an API error: {0}'.format(data['error'])
+        )
+    return data['dict']
+
+
+def show_cert(id_):
+    '''
+    Show certificate requests for this API key
+    '''
+    data = salt.utils.http.query(
+        '{0}/certificaterequests/{1}/certificate'.format(base_url, id_),
+        params={
+            'format': 'PEM',
+            'chainOrder': 'ROOT_FIRST'
+        },
+        status=True,
+        text=True,
+        header_dict={'tppl-api-key': _api_key()},
+    )
+    status = data['status']
+    if str(status).startswith('4') or str(status).startswith('5'):
+        raise CommandExecutionError(
+            'There was an API error: {0}'.format(data['error'])
+        )
+    data = data.get('body', '')
+    csr_data = salt.utils.http.query(
+        '{0}/certificaterequests/{1}'.format(base_url, id_),
+        status=True,
+        decode=True,
+        decode_type='json',
+        header_dict={'tppl-api-key': _api_key()},
+    )
+    status = csr_data['status']
+    if str(status).startswith('4') or str(status).startswith('5'):
+        raise CommandExecutionError(
+            'There was an API error: {0}'.format(csr_data['error'])
+        )
+    csr_data = csr_data.get('dict', {})
+    certs = _parse_certs(data)
+    dns_name = ''
+    for item in csr_data['certificateName'].split(','):
+        if item.startswith('cn='):
+            dns_name = item.split('=')[1]
+    #certs['CSR Data'] = csr_data
+
+    cache = salt.cache.Cache(__opts__, syspaths.CACHE_DIR)
+    domain_data = cache.fetch('venafi/domains', dns_name)
+    if domain_data is None:
+        domain_data = {}
+    certs['private_key'] = domain_data.get('private_key')
+    domain_data.update(certs)
+    cache.store('venafi/domains', dns_name, domain_data)
+
+    certs['request_id'] = id_
+    return certs
+
+
+pickup = show_cert
```

Comment:
IIRC, doesn't func alias make the old function name unavailable? I want both function names available here.